### DISCUSSION TITLE
Ch3nyang's blog | Python 指北 (2) - Python的import陷阱

### DISCUSSION THREAD

**Discussion by giscus** [WCY-dt/wcy-dt.github.io#21] [Comments]
# Ch3nyang's blog | Python的import陷阱

在 Python 中，当我们使用 import 语句导入模块时，Python 解释器会按照一定的规则查找模块：...

https://blog.ch3nyang.top/post/Python%E7%9A%84import%E9%99%B7%E9%98%B1/

**Comment by gebilxs** [comment]
nice catch

**Reply by WCY-dt** [reply]
thanks❤️
